core/src/main/java/org/geysermc/geyser/network/UpstreamPacketHandler.java

```diff
@@ -232,6 +232,9 @@ public PacketSignal handle(ResourcePackClientResponsePacket packet) {
                     stackPacket.getExperiments().add(new ExperimentData("data_driven_items", true));
                 }
 
+                // Required for experimental 1.21 features
+                stackPacket.getExperiments().add(new ExperimentData("updateAnnouncedLive2023", true));
```

Comment:
Does BDS send the experiment in both experiment lists (stackpacket and startgame) too?